tx.go

```diff
@@ -264,7 +264,13 @@ func (tx *Tx) rollback() {
 	}
 	if tx.writable {
 		tx.db.freelist.rollback(tx.meta.txid)
-		tx.db.freelist.reload(tx.db.page(tx.db.meta().freelist))
+		if !tx.db.hasSyncedFreelist() {
+			// Reconstruct free page list by scanning the DB.
```

Comment:
add a TODO here, why we need reload DB here, as reload entire DB costs (although etcd is rarely rollback)
)

Reply:
ok